statuspage.html

```diff
@@ -1,29 +1,29 @@
 <!DOCTYPE html>
-
-<html>
+
+<html>
   
-   <head>
+   <head>
      
       <meta http-equiv="Content-Type" content="text/html" charset="utf-8" lang="en">
-      <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0;">   

```

Comment:
maximum-scale=1.0, user-scalable=0 removed from meta content in the head tag of the html file.